netlib/utils.py

```diff
@@ -431,3 +430,16 @@ def safe_subn(pattern, repl, target, *args, **kwargs):
         need a better solution that is aware of the actual content ecoding.
     """
     return re.subn(str(pattern), str(repl), target, *args, **kwargs)
+
+def dequote(s):
```

Comment:
This is very similar to pathod's `inner_repr`. Can we combine these?
(cc @dufferzafar)

Also, tests please.